cli/src/commands/server-info.ts

```diff
@@ -0,0 +1,17 @@
+import { ServerVersionReponseDto } from 'src/api/open-api';
+import { BaseCommand } from '../cli/base-command';
+
+export default class ServerInfo extends BaseCommand {
+  static description = 'Display server information';
+  static enableJsonFlag = true;
+
+  public async run(): Promise<ServerVersionReponseDto> {
+    console.log('Getting server information');
+
+    await this.connect();
+    const { data: versionInfo } = await this.immichApi.serverInfoApi.getServerVersion();
+
+    console.log(versionInfo);
+    return versionInfo;
```

Comment:
Why do we need to log and return this information? Should it not be one of the other?

Reply:
Done